arm-devtestlabs/2016-05-15/swagger/DTL.json

```diff
@@ -2486,346 +2476,4367 @@
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevTestLab/labs/{labName}/virtualmachines/{name}/start": {
+            "post": {
+                "tags": [ "VirtualMachine" ],
+                "description": "Start a virtual machine. This operation can take a while to complete.",
+                "operationId": "VirtualMachine_Start",
+                "parameters": [
+                    { "$ref": "#/parameters/subscriptionId" },
+                    { "$ref": "#/parameters/resourceGroupName" },
+                    {
+                        "name": "labName",
+                        "in": "path",
+                        "description": "The name of the lab.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "description": "The name of the virtual machine.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    { "$ref": "#/parameters/api-version" }
+                ],
+                "responses": {
+                    "200": { "description": "OK" },
+                    "202": { "description": "Accepted" },
+                    "default": {
+                        "description": "BadRequest",
+                        "schema": { "$ref": "#/definitions/CloudError" }
+                    }
+                },
+                "security": [ { "oauth2": [ "user_impersonation" ] } ],
+                "x-ms-long-running-operation": true
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevTestLab/labs/{labName}/virtualmachines/{name}/stop": {
+            "post": {
+                "tags": [ "VirtualMachine" ],
+                "description": "Stop a virtual machine This operation can take a while to complete.",
+                "operationId": "VirtualMachine_Stop",
+                "parameters": [
+                    { "$ref": "#/parameters/subscriptionId" },
+                    { "$ref": "#/parameters/resourceGroupName" },
+                    {
+                        "name": "labName",
+                        "in": "path",
+                        "description": "The name of the lab.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "name",
+                        "in": "path",
+                        "description": "The name of the virtual machine.",
+                        "required": true,
+                        "type": "string"
+                    },
+                    { "$ref": "#/parameters/api-version" }
+                ],
+                "responses": {
+ 
```

Comment:
Or, are you saying that `name` in `CreateEnvironment` should be renamed to `labName` since it is not the name of the environment?